src/Components/Nav/Nav.js

```diff
@@ -1,41 +1,62 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { Link, withRouter } from 'react-router-dom';
 
 import UserLink from './UserLink';
 
 import './Nav.scss';
 
 class Nav extends React.Component {
+  isMain = () => {
+    const { location } = this.props;
+    return location.pathname === '/';
+  };
```

Comment:
굳이 함수로 만들 필요는 없을 것 같고 render 안에 변수로 만들어서 사용하는게 어떨까 싶습니다. 기능상 문제는 없을 것 같아 일단은 머지 하고, 이후 작업에서 반영해주세요 :)
```suggestion
  const isMain = this.props.location.pathname === '/';
```

Reply:
네 말씀해주신대로 변경하겠습니다.